frontend/src/components/dashboard/Dashboard.js

```diff
@@ -95,11 +101,15 @@ export class Dashboard extends Component {
         })
       );
     }
-    this.props.fetchStatusOptions();
-    this.props.fetchRegionOptions();
-    this.props.fetchMineTenureTypes();
-    this.props.fetchMineDisturbanceOptions();
-    this.props.fetchMineCommodityOptions();
+    if (!this.props.optionsLoaded) {
+      this.props.fetchStatusOptions();
+      this.props.fetchRegionOptions();
+      this.props.fetchMineTenureTypes();
+      this.props.fetchMineDisturbanceOptions();
+      this.props.fetchMineCommodityOptions();
+      this.props.fetchPartyRelationshipTypes();
+      this.props.setOptionsLoaded();
+    }
   }
```

Comment:
I really like this change. Good job!

What are your thoughts on these GET requests being moved to the highest level parent, ie Home.js, 
Then they can then be imported and called once, instead of in Dashboard.js and MineDashboard.js

Reply:
That's probably best. If I don't hit any snags in doing so I'll move it.

Reply:
I gave it a shot and it's not happy. I think it's related to authenticationguard on that component conflicting with connecting to the store. I've rolled my WIP changes back for now and would like to keep this as is as it's still a solid improvement.

Reply:
Understandable! Thank you for attempting.